src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -1055,7 +1056,9 @@ TEST(SpatialPoolerTest, testCalculateOverlap) {
                              {1, 1, 1, 1, 1}};
 
   for (UInt i = 0; i < numColumns; i++) {
-    vector<UInt> potential(permArr[i], permArr[i] + numInputs);
```

Comment:
Does this fail? Should be valid vector initialization

Reply:
> Does this fail? Should be valid vector initialization

There is an implied type conversion here with possible loss of data.  'permArr' is a Real and the destination array is UInt.  MSVC does not like this at all. Each layer of the template gives an error so I end up with a screen full of error message.  Actually it is a warning but I don't like to just suppress warnings.  I wish there was a better way to deal with this.